package.json

```diff
@@ -57,27 +57,26 @@
     "svgo": "^1.1.1"
   },
   "scripts": {
-    "setup": "npm run virtualenv && npm install && npm run config",
```

Comment:
no need for this anymore as we rely on .env
BUT I still need to write the doc for it.